Commands/HelpCommand.php

```diff
@@ -64,7 +67,7 @@ public function execute()
                 $data['text'] .= '/' . $user_command->getName() . ' - ' . $user_command->getDescription() . PHP_EOL;
             }
 
-            if (count($admin_commands) > 0) {
+            if ((count($admin_commands) > 0) && $safe_to_show) {
```

Comment:
Swap these, as a boolean is quicker to evaluate. If it's not safe to show anyway, no need to count the admin commands.

Small speed improvement, because every millisecond counts 😇 

(also, you can drop the parenthesis around `count(...) > 0`)